modules/oadp-kopia-algorithms-benchmarking.adoc

```diff
@@ -0,0 +1,177 @@
+Benchmarking hash 'HMAC-SHA3-256' (100 x 1048576 bytes, parallelism 1)
+     Hash                 Throughput
+-----------------------------------------------------------------
+  0. BLAKE3-256           15.3 GB / second
+  1. BLAKE3-256-128       15.2 GB / second
+  2. HMAC-SHA256-128      6.4 GB / second
+  3. HMAC-SHA256          6.4 GB / second
+  4. HMAC-SHA224          6.4 GB / second
+  5. BLAKE2B-256-128      4.2 GB / second
+  6. BLAKE2B-256          4.1 GB / second
+  7. BLAKE2S-256          2.9 GB / second
+  8. BLAKE2S-128          2.9 GB / second
+  9. HMAC-SHA3-224        1.6 GB / second
+ 10. HMAC-SHA3-256        1.5 GB / second
+-----------------------------------------------------------------
+Fastest option for this machine is: --block-hash=BLAKE3-256
+----
++
+. To benchmark the encryption algorithm, run the following command:
++
+[source,terminal]
+----
+sh-5.1# kopia benchmark encryption
+----
++
+.Example output
+[source,terminal]
+----
+Benchmarking encryption 'AES256-GCM-HMAC-SHA256'... (1000 x 1048576 bytes, parallelism 1)
+Benchmarking encryption 'CHACHA20-POLY1305-HMAC-SHA256'... (1000 x 1048576 bytes, parallelism 1)
+     Encryption                     Throughput
+-----------------------------------------------------------------
+  0. AES256-GCM-HMAC-SHA256         2.2 GB / second
+  1. CHACHA20-POLY1305-HMAC-SHA256  1.8 GB / second
+-----------------------------------------------------------------
+Fastest option for this machine is: --encryption=AES256-GCM-HMAC-SHA256
+----
++
+. To benchmark the splitter algorithm, run the following command:
++
+[source,terminal]
+----
+sh-5.1# kopia benchmark splitter
+----
++
+.Example output
+[source,terminal]
+----
+splitting 16 blocks of 32MiB each, parallelism 1
+DYNAMIC                     747.6 MB/s count:107 min:9467 10th:2277562 25th:2971794 50th:4747177 75th:7603998 90th:8388608 max:8388608
+DYNAMIC-128K-BUZHASH        718.5 MB/s count:3183 min:3076 10th:80896 25th:104312 50th:157621 75th:249115 90th:262144 max:262144
+DYNAMIC-128K-RABINKARP      164.4 MB/s count:3160 min:9667 10th:80098 25th:106626 50th:162269 75th:250655 90th:262144 max:262144
+# ...
+FIXED-512K                  102.9 TB/s count:1024 min:524288 10th:524288 25th:524288 50th:524288 75th:524288 90th:524288 max:524288
+FIXED-8M                    566.3 TB/s count:64 min:8388608 10th:8388608 25th:8388608 50th:8388608 75th:8388608 90th:8388608 max:8388608
+-----------------------------------------------------------------
+  0. FIXED-8M                  566.3 TB/s   count:64 min:8388608 10th:8388608 25th:8388608 50th:8388608 75th:8388608 90th:8388608 max:8388608
+  1. FIXED-4M                  425.8 TB/s   count:128 min:4194304 10th:4194304 25th:4194304 50th:4194304 75th:4194304 90th:4194304 max:4194304
+  # ...
+ 22. DYNAMIC-128K-RABINKARP    164.4 MB/s   count:3160 min:9667 10th:80098 25th:106626 50th:162269 75th:250655 90th:262144 max:262144
+----
```

Comment:
Maybe end the file with a blank line? Sometimes the customer portal can be funny with files with no empty space that impact the heading from rendering in the next file.